packages/core/README.md

```diff
@@ -392,6 +392,28 @@ server.tool("status", {
 });
 ```
 
+### Tool with Metadata
+
+Add `title` and `_meta` fields to pass arbitrary metadata through `tools/list` responses.
```

Comment:
nit: also passes through to tool call result https://modelcontextprotocol.io/specification/2025-06-18/schema#calltoolresult